### PR TITLE
Added mongodb steps

### PR DESCRIPTION
**Pull request by kenanjasim**

### CODE REVIEW

File: options.go [hunk 1]
```diff
@@ -302,6 +322,7 @@ func WithTestData(fsys fs.FS) TestSuiteOptionFunc {
 //	. . .
 //	cucumber.NewSuite("test", cucumber.WithHTTPData(httpData))
 func WithHTTPData(fsys fs.FS) TestSuiteOptionFunc {
+	fmt.Printf("http data: %+v\n", fsys)
```
Comment (tigh-latte) [inline options.go:325]: stray debug line

File: options.go [hunk 2]
```diff
@@ -310,6 +331,25 @@ func WithHTTPData(fsys fs.FS) TestSuiteOptionFunc {
...
+//
+// Usage example (using `embed.FS`):
+//
+//	//go:embed mongo/*
+//	var mongoData embed.FS
+//	. . .
+//	cucumber.NewSuite("test", cucumber.WithMongoData(httpData))
+func WithMongoData(fsys fs.FS) TestSuiteOptionFunc {
+	return func(t *testSuiteOpts) {
+		fmt.Printf("mongo data: %+v\n", fsys)
```
Comment (tigh-latte) [inline options.go:345]: stray debug line

**Review by tigh-latte — COMMENTED** [review]
Great feature! Thanks for the contribution.

Just a couple of stray debug lines to remove.

**Comment by kenanjasim** [comment]
> Great feature! Thanks for the contribution.
> 
> Just a couple of stray debug lines to remove.

I think in my tiredness yesterday I opened this MR to your repo instead of to the main in my fork 🙈

 I still have a few steps left so its still WIP 😄

**Pull request closed without merging**